torch/fx/passes/dialect/common/cse_pass.py

```diff
@@ -1,28 +1,27 @@
 from typing import Dict, Tuple, Any
 
 import torch
-import torch.fx as fx
 from torch.fx.passes.infra.pass_base import PassBase, PassResult
 from torch.utils._pytree import tree_flatten
 
-aten = torch.ops.aten
+from torch.fx import GraphModule, Graph
+from torch.fx import node as fxnode
```

Comment:
you can just do 

from torch.fx import Node

Reply:
thanks, fixed!